client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java

```diff
@@ -45,22 +45,31 @@ public class RssMRUtils {
 
   // Class TaskAttemptId have two field id and mapId, rss taskAttemptID have 21 bits,
   // mapId is 19 bits, id is 2 bits.
-  public static long convertTaskAttemptIdToLong(TaskAttemptID taskAttemptID) {
+  public static long convertTaskAttemptIdToLong(TaskAttemptID taskAttemptID, int appAttemptId) {
```

Comment:
update comment to describe how to get taskId

Reply:
done